src/ontology/lexmatch/README.md

```diff
@@ -13,14 +13,14 @@
  * Number of mappings in `unmapped_ordo_mondo`: 1
  * Number of mappings in `unmapped_doid_mondo`: 51
  * Number of mappings in `unmapped_ncit_mondo`: 3016
- * Number of mappings in `mondo_exactmatch_ncit`: 3543
- * Number of mappings in `mondo_closematch_ncit`: 224
- * Number of mappings in `mondo_broadmatch_ncit`: 17
- * Number of mappings in `mondo_exactmatch_doid`: 1448
- * Number of mappings in `mondo_closematch_doid`: 755
- * Number of mappings in `mondo_broadmatch_doid`: 25
- * Number of mappings in `mondo_narrowmatch_doid`: 26
- * Number of mappings in `mondo_exactmatch_icd10cm`: 2007
+ * Number of mappings in `mondo_narrowmatch_icd10cm`: 398
  * Number of mappings in `mondo_closematch_icd10cm`: 447
+ * Number of mappings in `mondo_exactmatch_icd10cm`: 2007
  * Number of mappings in `mondo_broadmatch_icd10cm`: 78
- * Number of mappings in `mondo_narrowmatch_icd10cm`: 398
+ * Number of mappings in `mondo_narrowmatch_doid`: 26
+ * Number of mappings in `mondo_closematch_doid`: 755
+ * Number of mappings in `mondo_exactmatch_doid`: 1448
+ * Number of mappings in `mondo_broadmatch_doid`: 25
+ * Number of mappings in `mondo_closematch_ncit`: 224
+ * Number of mappings in `mondo_exactmatch_ncit`: 3543
+ * Number of mappings in `mondo_broadmatch_ncit`: 17
```

Comment:
Whatever is writing this readme should be deterministic in the order of what is written, can you make a separate issue for that?

Reply:
Done[https://github.com/monarch-initiative/mondo-ingest/issues/177](url)